frontend/viewer/src/lib/entry-editor/object-editors/EntryEditor.svelte

```diff
@@ -147,12 +153,12 @@
   const currentView = useCurrentView();
 </script>
 
-<Editor.Root>
+<Editor.Root bind:ref>
   <Editor.Grid bind:ref={editorElem}>
     <EntryEditorPrimitive bind:entry {readonly} {modalMode} onchange={(entry) => onchange?.({entry})} />
 
     {#each entry.senses as sense, i (sense.id)}
-      <Editor.SubGrid class={cn(sense.id === highlightedEntity?.id && 'highlight')}>
+      <Editor.SubGrid class={cn(sense.id === highlightedEntity?.entity.id && 'highlight')}>
```

Comment:
should we maybe change the `highlightedEntity` name to just `highlighted` now that it has an entity property on it?